### PR TITLE
Update Buffered Host-Device Streaming FPGA readme

### PR DESCRIPTION
**Pull request by jkinsky**
# Existing Sample Changes
## Description

Restructured the readme to match the new template. Updated formatting. Updated some branding. Corrected some spelling and grammar. Moved images into /images folder to help clean up top-level folder. Expanded information related to running on DevCloud. Significantly restructured the content to move the build instruction to the top.

Fixes Issue# 

## External Dependencies

N/A

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [X] Documentation changes

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Command Line
- [x] VSCode

### CODE REVIEW

**Comment by jkinsky** [comment]
Adding @jimmytwei for notices.

File: DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/buffered_host_streaming/README.md
Comment (yuguen-intel) [inline DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/buffered_host_streaming/README.md:317]: This is the first time that you put the explanations of the sample after the results are shown right?
Do you think that this is a better template?

File: DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/buffered_host_streaming/README.md
Comment (yuguen-intel) [inline DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/buffered_host_streaming/README.md:321]: Same remark as in https://github.com/oneapi-src/oneAPI-samples/pull/1219 
I think that we should keep all the (FPGA) samples consistent and use the same naming convention for the folders
So to align with https://github.com/oneapi-src/oneAPI-samples/pull/1208 can you rename the `images` folder to `assets`, or disagree on the name and in that case we'll make all of these folders named `images` if you think that this is better

**Review by yuguen-intel — CHANGES_REQUESTED** [review]

File: DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/buffered_host_streaming/README.md
Comment (jkinsky) [reply DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/buffered_host_streaming/README.md:321]: > Same remark as in #1219 I think that we should keep all the (FPGA) samples consistent and use the same naming convention for the folders So to align with #1208 can you rename the `images` folder to `assets`, or disagree on the name and in that case we'll make all of these folders named `images` if you think that this is better

I updated this sample before I saw your response to change the name. I agree. Keep all samples for an area, like FPGA, as consistent as possible. Use the "assets" name for your images; I made similar changes using the "assets" folder name in later samples.

File: DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/buffered_host_streaming/README.md
Comment (jkinsky) [reply DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/buffered_host_streaming/README.md:317]: I think in this one case, the tutorial information is better placed below the build instructions. That's the recommendation for other tutorials.

File: DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/buffered_host_streaming/README.md
Comment (yuguen-intel) [reply DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/buffered_host_streaming/README.md:321]: I think that you need to push your changes then as this review still shows `images` :)

**Comment by jkinsky** [comment]
Amended the original commit to include the change from "images" to "assets".

**Review by yuguen-intel — APPROVED** [review]

**Comment by jimmytwei** [comment]
@jkinsky @yuguen-intel Any further changes to this PR? OK to merge?

**Comment by jkinsky** [comment]
> @jkinsky @yuguen-intel Any further changes to this PR? OK to merge?

I'm okay with merging.

**Comment by yuguen-intel** [comment]
I was waiting on @ishaghosh27 approval but if you are both ok with merging then I'm merging the change

**Pull request merged**
